app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str):
+    command_list = command.split(" ")
+    copy_from = command_list[1]
+    copy_to = command_list[2]
+
+    if copy_from != copy_to:
```

Comment:
also, you should check the input command `cp`

Reply:
ok, thank you)!